src/index.js

```diff
@@ -173,7 +173,13 @@ async function run() {
       await git.createTag(gitTag)
 
       core.info('Push all changes')
-      await git.push()
+      try {
+        await git.push()
+
+      } catch (error) {
+        console.error(error)
+        core.setFailed(error)
```

Comment:
Should we not add a `return` here so we prevent the code below from being executed?